newIDE/app/src/SceneEditor/index.js

```diff
@@ -1709,6 +1710,36 @@ export default class SceneEditor extends React.Component<Props, State> {
       this.editorDisplay.forceUpdateInstancesPropertiesEditor();
   };
 
+  forceUpdateCustomObjectRenderedInstances = () => {
+    const { project, projectScopedContainersAccessor } = this.props;
+
+    const resourcesInUse = new gd.ResourcesInUseHelper(
+      project.getResourcesManager()
+    );
+    projectScopedContainersAccessor.forEachObject(object => {
+      if (project.hasEventsBasedObject(object.getType())) {
+        object.getConfiguration().exposeResources(resourcesInUse);
+      }
+    });
+    const objectResourceNames = resourcesInUse
+      .getAllImages()
+      .toNewVectorString()
+      .toJSArray();
+    resourcesInUse.delete();
+
+    PixiResourcesLoader.loadTextures(project, objectResourceNames).then(() => {
+      const { editorDisplay } = this;
+      if (!editorDisplay) {
+        return;
+      }
+      projectScopedContainersAccessor.forEachObject(object => {
```

Comment:
Write a comment telling that all custom objects are reset (not just those with resources, it's something different)